### PR TITLE
updated access info and fixed formatting issues

### PR DESCRIPTION
**Pull request by lennessyy**
- Fixed formatting issue
- Added note about gzip file formats
- Updated reference regarding privilege and access

### CODE REVIEW

File: modules/data-loading/pages/s3-loader-user-guide.adoc [hunk 1]
```diff
@@ -63,75 +52,65 @@ This SET command reads, validates, and applies the configuration file, integrati
...
-
-==== ADVANCED: MultiGraph Support
+==== Global and local data sources
 
 The S3 Loader supports the TigerGraph MultiGraph feature. In the MultiGraph context, a data source can be either global or local:
 
-. A global data source can only be created by a superuser, who can grant the global data source to any graph.
-. An admin user can only create a local data source, which cannot be accessed by other graphs.
+* A global data source can only be created by a user with the `WRITE_DATASOURCE` privilege on the global scope, who can grant the global data source to any graph.
+* An user with `WRITE_DATASOURCE` privilege on a graph can only create local data sources on that graph, which cannot be accessed by other graphs.
```
Comment (a-m-thomas) [inline modules/data-loading/pages/s3-loader-user-guide.adoc:81]: a user with

Consistency in "with the X privilege" vs "with X privilege"

I think the second sentence sounds like the graph cannot be accessed by other graphs. Maybe break it into two sentences?

File: modules/data-loading/pages/s3-loader-user-guide.adoc [hunk 2]
```diff
@@ -150,70 +129,75 @@ DROP DATA_SOURCE *
...
-# The graph scope will only show the data source it has access to.
+// The global data source will be shown in global scope.
+// The graph scope will only show the data source it has access to.
 ----
 
-=== 2. Create a Loading Job
+=== 2. Create a loading job
 
-The S3 Loader uses the same basic https://docs.tigergraph.com/dev/gsql-ref/ddl-and-loading/creating-a-loading-job#create-loading-job[CREATE LOADING JOB] syntax used for standard GSQL loading jobs. A DEFINE FILENAME statement should be used to assign a loader FILENAME variable to a S3 data source name and the path to its config file.
+The S3 Loader uses the same basic xref:gsql-ref:ddl-and-loading:creating-a-loading-job.adoc[CREATE LOADING JOB] syntax used for standard GSQL loading jobs.
```
Comment (a-m-thomas) [inline modules/data-loading/pages/s3-loader-user-guide.adoc:149]: Do we want backticks to get monospacing even when CREATE LOADING JOB is a hyperlink? I could go either way but this is something we should be consistent on and I can't find another example one way or another.

File: modules/data-loading/pages/s3-loader-user-guide.adoc [hunk 3]
```diff
@@ -150,70 +129,75 @@ DROP DATA_SOURCE *
...
 DEFINE FILENAME f1 = "$s1:~/files.config";
 ----
 
-==== S3 File Configuration File
+==== S3 file configuration file
 
-The S3 file configuration file tells the TigerGraph system exactly which Amazon S3 files to read and how to read them. Similar to the data source configuration file described above, the contents are in JSON object format. An example file is shown below.
+The S3 file configuration file tells the TigerGraph system exactly which Amazon S3 files to read and how to read them.
+Similar to the data source configuration file described above, the contents are in JSON object format.
+An example file is shown below.
```
Comment (a-m-thomas) [inline modules/data-loading/pages/s3-loader-user-guide.adoc:176]: Change "below" or remove

File: modules/data-loading/pages/s3-loader-user-guide.adoc [hunk 4]
```diff
@@ -150,70 +129,75 @@ DROP DATA_SOURCE *
...
 
-
-
-The "file.uris" key is required. It specifies one or more paths on your Amazon S3 bucket. Each path is either to an individual file or to a directory. If it is a directory, then each file directly under that directory is included. You can specify multiple paths by using a comma-separated list. An example with multiple paths is show below:
+The `file.uris` key is required.
+It specifies one or more paths on your Amazon S3 bucket.
+Each path is either to an individual file or to a directory.
+If it is a directory, then each file directly under that directory is included.
+You can specify multiple paths by using a comma-separated list.
+An example with multiple paths is show below:
```
Comment (a-m-thomas) [inline modules/data-loading/pages/s3-loader-user-guide.adoc:191]: Change "below" or remove

File: modules/data-loading/pages/s3-loader-user-guide.adoc [hunk 5]
```diff
@@ -222,9 +206,10 @@ DEFINE FILENAME f1 = "$s1:~/files.config";
 DEFINE FILENAME f1 = "$s1:{\"file.uris\":\"s3://my-bucket/data.csv\"}";
 ----
 
-==== ADVANCED: Configure How to Read S3 File
+==== Configure how to read S3 file
```
Comment (a-m-thomas) [inline modules/data-loading/pages/s3-loader-user-guide.adoc:209]: Configure S3 file reading (loading?)

File: modules/data-loading/pages/s3-loader-user-guide.adoc [hunk 6]
```diff
@@ -333,19 +339,19 @@ USE GRAPH test_graph
 DROP JOB load_person
 DROP DATA_SOURCE s1
 
-# Create data_source s3 s1 = "s3_config.json" for graph test_graph.
-CREATE DATA_SOURCE S3 s1 FOR GRAPH test_graph
+// Create data_source s3 s1 = "s3_config.json" for graph test_graph.
+CREATE DATA_SOURCE S3 s1 FOR GRAPH est_graph
```
Comment (a-m-thomas) [inline modules/data-loading/pages/s3-loader-user-guide.adoc]: test_graph

File: modules/data-loading/pages/s3-loader-user-guide.adoc (hunk 2 above)
Comment (lennessyy) [reply modules/data-loading/pages/s3-loader-user-guide.adoc:149]: Yes, we should always put backticks over code. I have added them.

**Pull request merged**